### ISSUE TITLE
How to find the default definition of some parameters in the nmt model?

### ISSUE THREAD

**Issue by yapingzhao** [tensorflow/nmt#301]
Hi,
First of all, I'm a bit embarrassed, because my problem below may be a bit too simple. Let me explain first. I'm a neural network beginner (smile).
I have several questions that I haven't found, so I need help.
1. I didn't find the default defined word vector size (in the model.py document).
2. I didn't find out about the activation function inside the neural network.
3, learning rate of the model training, if the parameter optimization sgd selection, learning rate changes (attenuation); if the parameter optimization adam, learning rate changes (attenuation)
4. Finally, what algorithm is the default for the decoder's search algorithm, the nmt model? When does the search algorithm work?
Thank you for reading my question.
Looking forward to your advice or answers.
Best regards,

yapingzhao
